test/system/maintenance_tasks/runs_test.rb

```diff
@@ -72,6 +77,26 @@ class RunsTest < ApplicationSystemTestCase
       assert_text "Waiting to start."
     end
 
+    test "run a CSV Task with direct upload" do
+      with_direct_upload(true) do
+        visit maintenance_tasks_path
+
+        click_on("Maintenance::ImportPostsTask")
+
+        attach_file("csv_file", "test/fixtures/files/sample.csv")
+        click_on "Run"
+        upload_url = page.server_url + "/rails/active_storage/direct_uploads"
+
+        assert_selector("input[type=file]") do |input|
+          assert_equal(input["data-direct-upload-url"], upload_url)
```

Comment:
`input["data-direct-upload-url"]` is returning nil for 'Rails <=7.0'

More specifically, it is failing here from the looks of it: https://github.com/rails/rails/blob/c11312a918555aaad99b334d374a791cd6268152/actionview/lib/action_view/helpers/form_tag_helper.rb#L1078